pages/network/[network]/index.tsx

```diff
@@ -69,13 +70,11 @@ export default function Home({ loans }: HomeProps) {
 
   return (
     <>
-      <Head>
-        <title>Backed | Home</title>
-        <meta
-          name="description"
-          content="Welcome to Backed protocol. View existing loans, lend against NFTs, or propose loan terms on your own NFTs."
-        />
-      </Head>
+      <OpenGraph
+        title="Backed Protocol"
```

Comment:
For the titles let's keep the `Backed | <network> | <page description>` pattern as seen [here](https://github.com/with-backed/backed-interface/blob/main/pages/network/%5Bnetwork%5D/loans/%5Bid%5D.tsx#L196)

Reply:
the latest commits should resolve this